braid/_braid.c

```diff
@@ -555,9 +554,15 @@ _braid_AdjointFeatureCheck(braid_Core core)
  * Macros used below
  *----------------------------------------------------------------------------*/
 
+#define _braid_SendIndexOff -2
+#define _braid_RecvIndexOff -2
+
+#define _braid_MapPeriodic(index, npoints)                              \
+   ( index = ((index)+(npoints)) % (npoints) )  /* this also handles negative indexes */
```

Comment:
@rfalgout 
Isn't 
( K + M ) % M == K % M
?
That is, couldn't we do 
index = index % npoints
?